### PR TITLE
Feature/comp layout

### PR DESCRIPTION
**Pull request by michelecomfort**
**What (if any) features are you implementing?**
 - Grid for widget cards to start building out responsiveness
 
**What (if anything) did you refactor?**
 - Moved charts to separate file
 - Removed css code that didn't have purpose
 
**Were there any issues that arose?**
-

**Is there anything that needs to be prioritized?**
 - connecting Activity to DOM
 - Cloning down data for GET/POST requests

 
**Next Steps**
  - Media Breakpoints

### CODE REVIEW

**Review by Markus-Xavier — APPROVED** [review]
Code looks great

File: src/css/styles.css [hunk 1]
```diff
@@ -119,22 +124,42 @@ body,
...
-.water,
-.steps,
-.sleep,
-.friends,
-.today-data,
-.user-data,
-.step-goals,
-.sleep-card {
+.sidebar {
+  position: -webkit-sticky;
```
Comment (kbergsund) [inline src/css/styles.css:133]: super nifty, thanks for considering cross-browser functionality

File: src/css/styles.css [hunk 2]
```diff
@@ -119,22 +124,42 @@ body,
...
+  position: sticky;
+  top: 0;
+}
+
+.widgets {
```
Comment (kbergsund) [inline src/css/styles.css:138]: Yay grid!

File: src/css/styles.css [hunk 3]
```diff
@@ -281,4 +290,15 @@ body,
...
   margin-top: 7%;
-}
\ No newline at end of file
+}
+
+.steps-miles {
+  width: 80%;
+}
+
+@media screen and (min-width: 1990px) {
```
Comment (kbergsund) [inline src/css/styles.css:299]: Amazing, looking forward to making it even more responsive.

**Review by kbergsund — APPROVED** [review]

**Pull request merged**